next_steps.md

```
@@ -0,0 +1,301 @@
+1. **Compusophia JARTIC Archives** (PRIMARY SOURCE)
+   - Historical general road traffic data from 2023-2025
+   - 5-minute and hourly intervals
+   - Covers ~2,600 observation points nationwide
+   - Free download of archived monthly zip files
+   - URL: https://www.compusophia.com/en/notes/1
+
+2. **JARTIC Open Traffic API** (SUPPLEMENTARY)
+   - Real-time + 3 months historical data
+   - Requires registration but free to use
+   - Same observation points as archives
+   - URL: https://www.jartic-open-traffic.org/
```

markdownlint-cli2 (0.17.2)

Bare URL used

(MD034, no-bare-urls)